### PR TITLE
Inline all consts

### PR DESCRIPTION
**Pull request by mfojtik**
Using consts everywhere makes it impossible to grep the code for strings. These strings will never change and if they do it will be when we will rewrite this operator.

/cc @enj 
/cc @deads2k

### CODE REVIEW

**Comment by deads2k** [comment]
/lgtm

We'll start here and I'll take pulls going in the other direction for "openshift-config", "openshift-config-managed" (library-go is appropriate) and the other operator namespaces in a `pkg/operator/operatorclient` package like the rest.

**Comment by enj** [comment]
(comments coming)

File: pkg/operator2/route.go [hunk 1]
```diff
@@ -141,5 +141,5 @@ func isIngressAdmitted(ingress routev1.RouteIngress) bool {
 }
 
 func ingressToHost(ingress *configv1.Ingress) string {
-	return targetName + "." + ingress.Spec.Domain
+	return ("oauth-openshift") + "." + ingress.Spec.Domain
```
Comment (enj) [inline pkg/operator2/route.go]: fix

File: pkg/operator2/route.go [hunk 2]
```diff
@@ -85,8 +85,8 @@ func routerSecretToSNI(routerSecret *corev1.Secret) []configv1.NamedCertificate
 		out = append(out, configv1.NamedCertificate{
 			Names: []string{"*." + domain}, // ingress domain is always a wildcard
 			CertInfo: configv1.CertInfo{ // the cert and key are appended together
-				CertFile: routerCertsLocalMount + "/" + domain,
-				KeyFile:  routerCertsLocalMount + "/" + domain,
+				CertFile: ("/var/config/system/secrets/v4-0-config-system-router-certs") + "/" + domain,
```
Comment (enj) [inline pkg/operator2/route.go]: fix

File: pkg/operator2/resourceversion_test.go [hunk 3]
```diff
@@ -32,70 +32,70 @@ func Test_authOperator_handleConfigResourceVersions(t *testing.T) {
...
 		{
 			name: "both config overlapping resource versions and ignored data",
 			objects: []runtime.Object{
 				testRVSecret("e", "5"),
 				testRVConfigMap("f", "6"),
-				testRVSecret(userConfigPrefix+"a", "3"),
-				testRVConfigMap(userConfigPrefix+"b", "2"),
-				testRVSecret(systemConfigPrefix+"c", "2"),
-				testRVConfigMap(systemConfigPrefix+"d", "3"),
+				testRVSecret(("v4-0-config-user-")+"a", "3"),
```
Comment (enj) [inline pkg/operator2/resourceversion_test.go]: fix all

File: pkg/operator2/deployment.go [hunk 4]
```diff
@@ -112,25 +112,25 @@ func defaultDeployment(
...
+							Name:            "oauth-openshift",
 							Command:         []string{"/bin/bash", "-ec"},
 							Args: []string{fmt.Sprintf(`
 if [ -s %s ]; then
     echo "Copying system trust bundle"
     cp -f %s /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
 fi
 exec oauth-server osinserver --config=%s --v=%d
-`, trustedCABundlePath, trustedCABundlePath, cliConfigPath, getLogLevel(operatorConfig.Spec.LogLevel))},
+`, "/var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle/ca-bundle.crt", "/var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle/ca-bundle.crt", "/var/config/system/configmaps/v4-0-config-system-cliconfig/v4-0-config-system-cliconfig", getLogLevel(operatorConfig.Spec.LogLevel))},
```
Comment (enj) [inline pkg/operator2/deployment.go]: put inside the raw string

File: pkg/operator2/configsync_test.go [hunk 5]
```diff
@@ -304,65 +304,65 @@ func Test_authOperator_handleConfigSync(t *testing.T) {
...
-				testConfigSyncConfigMap(userConfigPrefix + "dest-0"),
-				testConfigSyncConfigMap(userConfigPrefix + "dest-1"),
+				testConfigSyncConfigMap(("v4-0-config-user-") + "dest-0"),
+				testConfigSyncConfigMap(("v4-0-config-user-") + "dest-1"),
 
-				testConfigSyncSecret(userConfigPrefix + "dest-2"),
-				testConfigSyncSecret(userConfigPrefix + "dest-3"),
-				testConfigSyncSecret(userConfigPrefix + "dest-4"),
-				testConfigSyncSecret(userConfigPrefix + "dest-5"),
+				testConfigSyncSecret(("v4-0-config-user-") + "dest-2"),
```
Comment (enj) [inline pkg/operator2/configsync_test.go]: lots more fix

File: pkg/operator2/operator.go [hunk 6]
```diff
@@ -41,106 +41,21 @@ import (
...
-	rootCAFile = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-
-	configVersionPrefix = "v4-0-config-"
-
-	// secrets and config maps that we manually managed have this prefix
-	systemConfigPrefix = "v4-0-config-system-"
-
-	// secrets and config maps synced from openshift-config into our namespace have this prefix
-	userConfigPrefix = "v4-0-config-user-"
-	// idps that are synced have this prefix
```
Comment (enj) [inline pkg/operator2/operator.go]: comments like these should be preserved since they are not obvious

File: pkg/operator2/idp.go [hunk 7]
```diff
@@ -17,34 +17,8 @@ import (
...
-
-	bindPasswordField = "bind-password"
-
-	loginField             = "login"
-	providerSelectionField = "provider-selection"
-	errorField             = "error"
-)
-
 var (
+	// field names are used to uniquely identify a secret or config map reference
```
Comment (enj) [inline pkg/operator2/idp.go]: I want the whole paragraph not just the first line :|

**Comment by enj** [comment]
@mfojtik almost.

File: pkg/operator2/idp.go (hunk 7 above)
Comment (mfojtik) [reply pkg/operator2/idp.go]: copied it back

**Pull request merged**